NUnitTests/Div_Neg_Test.cs

```diff
@@ -0,0 +1,28 @@
+using CSharpCalculator;
+using NUnit.Framework.Constraints;
+
+namespace NUnitTests;
+
+public class Div_Neg_Test //division by zero //bug
```

Comment:
You have two classes to test Divide method. So I recommend to combine this tests to one class.